server/controller/visitors.js

```diff
@@ -0,0 +1,32 @@
+const { Visitor } = require('../models');
```

Comment:
## Nitpick

Resources should be named in plural, ie `controllers` not `controller`